frontend/components/icons/AppleCircledGrey.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { ICON_SIZES, IconSizes } from "styles/var/icon_sizes";
+
+interface IAppleGreyProps {
+  size: IconSizes;
+}
+const AppleCircledGrey = ({ size = "extra-large" }: IAppleGreyProps) => {
```

Comment:
Instead of making 4 new icons, can we just replace the colors of the existing ones that are being replaced?

We can make it work for both colors if we want:

Example of combining the existing `ApplePurple.tsx` with this `AppleCircledGrey.tsx` with grey as the default:
<img width="1222" alt="Screenshot 2023-07-28 at 10 25 02 AM" src="https://github.com/fleetdm/fleet/assets/71795832/c05a9dfa-1363-416d-b8cb-c4a019aa7899">

Reply:
https://github.com/RachelElysia/fleet/commit/0d2ea780ca1045b6e1f19ddf7ac97ddd21397265

Reply:
@RachelElysia I agree that here and in general, building with modularity and extensibility in mind is ideal. Since this was a minor task and so didn’t warrant too much developer time/energy, and in case there had been some previous discussions around this I was unaware of, I decided to follow the pattern you’d already established in the code: `ApplePurple`, `WindowsBlue`, etc.

If you feel it’s worth the dev capacity, since this is already done and it seems you’ve gotten started on a refactor, can you approve this and then push up your refactor as a follow-up PR?

Reply:
Are the old ones being used anywhere? What's our FE philosophy on keeping unused code?

Reply:
Are the old ones being used anywhere? What's our FE philosophy on keeping unused code?